test/benchmarks/README.md

```diff
@@ -1,15 +1,38 @@
-install google benchmark: https://github.com/google/benchmark
+# VW Benchmarks
 
-by default google benchmark is built in Debug mode so you might want to specify Release mode when building/installing
+## Benchmark results
+Automated benchmarking is performed as part of the VW continuous integration system. Benchmark results are available here: **https://vowpalwabbit.org/vowpal_wabbit/dev/bench/**
 
-build vw benchmarks:
+Benchmarking is done via [github-action-benchmark](https://github.com/benchmark-action/github-action-benchmark).
 
+## Running benchmarks locally
+### C++
+Compile:
 ```
-make -j vw-benchmarks.out
+cmake --preset vcpkg-release -DBUILD_BENCHMARKS=On
+cmake --build build --target vw-benchmarks.out
 ```
 
-run vw benchmarks:
+Run:
+```
+./build/test/benchmarks/vw-benchmarks.out
+```
+
+### .NET
+First, install the VW Nuget packages.
```

Comment:
nit: do we have documentation for this or a location that we can link to?

Reply:
Unfortunately not that I know of